### PR TITLE
docs: update data viz docs with sparklines

### PR DESCRIPTION
**Pull request by didoesdigital**
This PR adds sparkline guidance.

Branch preview: https://dev.cultureamp.design/di/update-data-viz-docs/guidelines/data-visualization/

image

### CODE REVIEW

File: site/docs/guidelines/data-visualization.mdx [hunk 1]
```diff
@@ -123,6 +127,28 @@ Learn more about bar charts:
 - [Morningstar: horizontal bar chart](https://designsystem.morningstar.com/charts/horizontal-bar.html)
 - [Government of Canada, Aurora: Bar charts](https://design.gccollab.ca/data/column-and-bar-graphs)
 
+## Sparklines
+
+<iframe style="border: none;" width="744" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2Fe3TyAOZKfLg9cA01Iy1Mgm%2FKaizen-Site-embed%3Fnode-id%3D54258%253A0&chrome=DOCUMENTATION" allowfullscreen></iframe>
+
+A sparkline is a condensed, word-sized chart showing trend over time. It can be presented anywhere a word or number can, such as in a sentence, table, heading, graphic, or spreadsheet.
+
+Sparklines are usually shown without axes or labels to focus on the shape of data. Multiple sparklines are often presented together as elements of a small multiple. They're also often presented alongside hero data, showing the last value in the sparkline.
```
Comment (dlimiter) [inline site/docs/guidelines/data-visualization.mdx:136]: I'm finding this sentence a little hard to parse. 
- I don't understand what you mean by ''presented together as elements of a small multiple"?
- "They're also often presented alongside hero data, showing the last value in the sparkline." I got a bit turned around here as well as to what was showing what - like the hero data it there to add context to the sparkline rather than the other way around.  Is it too reductive to say "Sparklines are also be shown alongside hero data to provide additional context."?

**Review by sebpearce — APPROVED** [review]
